tensorflow/tensorflow.bzl

```diff
@@ -338,6 +344,7 @@ def tf_copts(
         if_android_arm(["-mfpu=neon"]) +
         if_linux_x86_64(["-msse3"]) +
         if_ios_x86_64(["-msse4.1"]) +
+        if_no_default_logger(["-DNO_DEFAULT_LOGGER"]) +
```

Comment:
Hmm, that may have been a mistake, I'll take a look.

Reply:
OK, I looked and that statement is not just about Android, though Android is one of the many options. As far as I can tell, this is the most appropriate place to set the preprocessor symbol.

Reply:
P.S. The reason I didn't document it there is because the option is already documented, while this statement just calls stuff, it doesn't document anything else. I thought a comment there would be out of place.